src/main/resources/db/migration/V1__create_client_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE `client` (
+    id int AUTO_INCREMENT NOT NULL,
```

Comment:
This cant be called id, it should be called client id. It makes it hard to read should we have a foreign key